### ISSUE TITLE
ResponseError with code UnkownError shows up once Activity starts

### ISSUE THREAD

**Issue by drori200** [JnCrMx/discord-game-sdk4j#56]
I've implemented and modified the activity example for my needs and so far it works great. Everything runs and behaves like it should.

Issue is, when starting up my RPC I receive these lines:

```
[ERROR] ResponseError { code: UnknownError, message: "Request has been terminated\nPossible causes: the network is offline, Origin is not allowed by Access-Control-Allow-Origin, the page is being unloaded, etc." }

[ERROR] Failed to configure networking: ResponseError { code: UnknownError, message: "Request has been terminated\nPossible causes: the network is offline, Origin is not allowed by Access-Control-Allow-Origin, the page is being unlo
aded, etc." }
```

The program doesn't crash and even still runs fine but this kind of concerns me. Below is my activity file:
https://gist.github.com/drori200/a61284adeffc26117f087aee045620ff

**Comment by letorbi** [comment]
I have experienced the same issue, when I use the current release (0.5.5), which uses the official Discord Games SDK binary. However, no `ResponseError` occurs, when I am using the *java-impl* branch instead.

The *java-impl* branch is a pure-java implementation, which does not use the Discord Games SDK binary internally. The branch is currently experimental, but it should be usable once PR #59 has been merged.

**Comment by Robatortas** [comment]
Yeah, I think it's a library issue, since it just suddenly stopped functioning for me too.

And also the error is not even as a System.err.println error, sooo I guess it's the library itself...
